### DISCUSSION TITLE
BB80E5EA-3BC9-31F5-9A34-C424011803F8

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#84] [General]
# V616 Monocerotis

V616 Monocerotis (A0620-00) is the third nearest black hole system to Earth, about 4,700 light-years away in the constellation Monoceros. In 2018-Jun-15, a radio signal was sent to this black hole in memory of the late Dr. Stephen Hawking, who died three months prior.

(NOTE: Black hole addons tend to be performance heavy and might not work in all devices)

https://celestia.mobi/addon?item=BB80E5EA-3BC9-31F5-9A34-C424011803F8&lang=en

**Comment by bubbyiscooling** [comment]
I WANT TO SEE IT IN GAME

**Reply by TheLostProbe** [reply]
then download it 💀💀💀

**Reply by LangLangFCS** [reply]
WHAT

**Comment by pjawesome2023** [comment]
It takes so fucking long

**Reply by TheLostProbe** [reply]
I mean it does say "NOTE: Black hole addons tend to be performance heavy and might not work in all devices"

**Reply by ilyushinil594-hue** [reply]
Bro he's tuff that lagging in his game

**Comment by exocritic5457** [comment]
I do wonder if the Gaia BH systems will ever make the cut.

**Reply by SevenSpheres** [reply]
[Gaia BH1](https://celestiaproject.space/forum/viewtopic.php?f=23&t=23159) was released, I think Gaia BH2 is only in my black hole catalog that was never finished

**Comment by Cardenasdidier23-max** [comment]
If black holes are lagging your game, just restart the game

**Comment by LangLangFCS** [comment]
Beautiful.Amazing.
